### ISSUE TITLE
is there a way to add a prompt to better guide the result?

### ISSUE THREAD

**Issue by Damojan** [DiffusionLight/DiffusionLight#24]
i made an hdri using DiffusionLight  of a bathroom but the room looks like a living room with a softbox instead of a large window with trees that its supposed to be, is there anyway to prompt the resut?

"hdri of a bathroom with white tiles marble vanity and a large window with trees outside"

**Comment by pureexe** [comment]
You can change the text prompt by providing `--prompt` and `--prompt_dark` to the inpaint.py

the default prompt is "a perfect mirrored reflective chrome ball sphere" and  "a perfect **black dark** mirrored reflective chrome ball sphere"

In your case you might want to try: 

```
python inpaint.py --dataset <input_directory> --output_dir <output_directory> --prompt "a perfect mirrored reflective chrome ball sphere in the bathroom with white tiles marble vanity and a large window with trees outside" -- prompt_dark "a perfect black dark mirrored reflective chrome ball sphere in the bathroom with white tiles marble vanity and a large window with trees outside"
```

**Comment by pureexe** [comment]
Yes, you change the code at **inpainting chromeball** block to what you have mentioned above.

**Comment by Damojan** [comment]
i tried adding that code in the collab and it gave me an error message.

**Comment by pureexe** [comment]
What's an error? Can you give me a screenshot?
